.teamcity/NativePlatformPublishing.kt

```diff
@@ -61,8 +61,6 @@ open class NativePlatformPublishSnapshot(releaseType: ReleaseType, uploadTasks:
     params {
         param("ARTIFACTORY_USERNAME", releaseType.username)
         password("ARTIFACTORY_PASSWORD", releaseType.password, display = ParameterDisplay.HIDDEN)
-        password("pgpSigningKey", "credentialsJSON:ae6de222-f77d-4c71-b94e-fac6da485143", display = ParameterDisplay.HIDDEN)
```

Comment:
The downside of moving the parameter further up is that all projects in Gradle Native can access is. Given how we handle the key elsewhere, I suppose that is fine.